modules/deployments-ab-testing-lb.adoc

```diff
@@ -54,6 +54,11 @@ Browse to the application at `ab-example-a.<project>.<router_domain>` to verify
 
 . When you deploy the route, the router balances the traffic according to the `weights` specified for the services. At this point, there is a single service with default `weight=1` so all requests go to it. Adding the other service as an `alternateBackends` and adjusting the `weights` brings the A/B setup to life. This can be done by the `oc set route-backends` command or by editing the route.
 +
+[NOTE]
+====
+When using `alternateBackends` also use the `roundrobin` load balancing strategy to ensure requests are distributed as expected to the services based on weight. `roundrobin` can be set for a route using a link:https://docs.openshift.com/container-platform/4.13/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration[route annotation].
```

Comment:
```suggestion
When using `alternateBackends`, also use the `roundrobin` load-balancing strategy to ensure requests are distributed as expected to the services based on weight. `roundrobin` can be set for a route by using a link:https://docs.openshift.com/container-platform/4.13/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration[route annotation].
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-after-introductory-words-and-phrases
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-hyphens#compound-words